File: CalculatorApp/CalculatorAppTests/Utils.Tests.Unit/CalculatorTests.m
#import "CalculatorTests.h"
#import "Calculator.h"

@implementation CalculatorTests {
    Calculator *_sut;
}

- (void)setUp {
    [super setUp];
    _sut = [[Calculator alloc] init];
}


-(void)testGivenIWantToAdd_whenZeroLengthInputIsPassedIn_thenNumericZeroShouldBeReturned {
    NSInteger expected = 0;
    NSInteger result = [_sut add:@""];

    STAssertEquals(expected, result, @"Zero length input should return 0.");
}
@end

File: CalculatorKataSwift/CalculatorKataSwiftTests/CalculatorTests.swift
import XCTest
import CalculatorKataSwift

class CalculatorTests: XCTestCase {

    
    override func setUp() {
        super.setUp()
    }

    func test_given_Sut_when_zeroLengthstringAdded_then_shouldReturn0() {
        var expected = 0
        var sut = Calculator()
        var actual = sut.add("")
        
        XCTAssertEqualObjects(expected, actual)
    }
    
    func test_given_Sut_when_oneLengthstringAdded_then_shouldReturnNumericEquivalent() {
        var expected = 3
        var sut = Calculator()
        var actual = sut.add("3")
        
        XCTAssertEqualObjects(expected, actual)
    }
    
}
